### PR TITLE
Require and Assert.

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/1762

### CODE REVIEW

File: docs/control-structures.rst [hunk 1]
```diff
@@ -396,12 +396,19 @@ Currently, Solidity automatically generates a runtime exception in the following
...
 
 While a user-provided exception is generated in the following situations:
 #. Calling ``throw``.
+#. Calling ``require`` with an argument that evaluates to ``false``.
 
-Internally, Solidity performs a revert operation (instruction ``0xfd``) when a user-provided exception is thrown. In contrast, it performs an invalid operation
-(instruction ``0xfe``) if a runtime exception is encountered. In both cases, this causes
+Internally, Solidity performs a revert operation (instruction ``0xfd``) when a user-provided exception is thrown or the condition of
+a ``require`` call is not met. In contrast, it performs an invalid operation
+(instruction ``0xfe``) if a runtime exception is encountered or the condition of an ``assert`` call is not met. In both cases, this causes. In both cases, this causes
```
Comment (pirapira) [inline docs/control-structures.rst]: "In both cases, this causes" is duplicated.

**Comment by pirapira** [comment]
Please rebase.

**Comment by chriseth** [comment]
Rebased.

**Review by pirapira — APPROVED** [review]
Looks good to me.

**Pull request merged**